src/payment/payment.ts

```diff
@@ -80,6 +82,10 @@ export interface PaypalInstrument {
     };
 }
 
+export interface BarclaycardInstrument {
+    browser_info: BarclaycardBrowserInfoRequest;
```

Comment:
Is there any reason why it needs to be tied with a specific payment method? It seems to me that it's just some generic information about the user's device. And some of the information the server can already derive from the user agent of a request so the client probably doesn't need to send them separately.

What I want to see is that the client sends the minimal amount of information that's not tied to a particular payment method. The server then receives the information and transforms it to the payload schema / format that the payment method expects. 

The problem with the current approach is that what happens if another payment method requires us to send browser information as well. But that one requires us to send it in a different format. But we can't because `browser_info` is tied with Barclaycard.